lib/wibox/widget/imagebox.lua

```diff
@@ -135,11 +135,24 @@ local function load_and_apply(ib, file, image_loader, image_setter)
     return image_applied
 end
 
---- Set an imagebox' image
+--- The image rendered by the `imagebox`.
+-- <br />It can can be any of the following:
+--
+-- * A `string` : Interpreted as the path to an image file ;
+-- * A cairo image surface : Directly used as is ;
+-- * An rsvg handle object : Directly used as is ;
+-- * `nil` : Deny previously setted image.
```

Comment:
I have basically no clue about ldoc, so: Is the `<br />` necessary? Are the `;` necessary?

Reply:
The `<br />` is basically used as an HTML tag to insert a carriage-return character. I used it to separate the summary from the body description of the method. Summary only is shown in the "methods table" at the beginning of the doc. Then in the detail of the method you have Summary+Body. I wanted them to be line separated here so I added this `<br />`.

It could also have be done with an empty line:

```lua
--- The image rendered by the `imagebox`.
-- 
-- It can can be any of the following:
```

The `;` is more an habit I have when doing lists. According to https://getitwriteonline.com/articles/vertical-lists/ I should rather use `,` or `.`, tho.